_sections/COVID-19/en/08-the-us-healthcare-system.md

```diff
@@ -1,9 +1,11 @@
 {:.no_toc}
-## The US healthcare system is not well enough prepared. No one is
+## No country is fully prepared, including the United States
 
-In the USA, our health care capacity is under 1 million staffed beds; this is not adequate to accommodate the number of [hospitalizations we
-are expected to see (4-8 million)](https://www.bloomberg.com/opinion/articles/2020-03-05/how-bad-is-the-coronavirus-let-s-compare-with-sars-ebola-flu). Johns Hopkins University did a 3-year global survey of pandemic preparedness. Although the USA ranked at the top of the countries, even the US only [scored a 42 out of 100.](https://jhu.pure.elsevier.com/en/publications/pandemic-influenza-and-major-disease-outbreak-preparedness-in-us--7) We simply don't have the equipment, training, or messaging in place and
```

Comment:
I obtained the complete paper from one of the authors, but it doesn't score the US as "42 out of 100". The only mention of 42 was:
> Only 42 percent of respondents felt that their ED was prepared in the event of a pandemic influenza or other disease outbreak, and only 35 percent felt that their hospital was prepared.

Instead, I opted to link to a much more recent report from the WHO per @steesdale's advice.

Reply:
The 42 number was one that struck me from a webinar that JHU did in early March. There must be a recording somewhere. In the interest of time, rather than say 42, let's just go with 'scored poorly' because even if I misremember the exact number that was stated in the webinar, the sentiment is correct.

Reply:
@jmcmurry: See [this](https://github.com/flattenthecurve/guide/pull/277/commits/4596f46a010c0c3c4a844ca5101fe542f60963d9) commit. I opted to re-add a link to the JHU study, but quoted it directly. Let me know if that works for you!